helm/hpcc/templates/_warnings.tpl

```diff
@@ -56,7 +56,7 @@ Pass in dict with root and warnings
   {{- $_ := set $ctx "warnings" (append $ctx.warnings $warning) -}}
  {{- end -}}
  {{- /* Warn if any storage planes uses default cost values */ -}}
- {{- if or (eq .root.Values.global.cost.storageWrites 0.0500000000001) (or (eq .root.Values.global.cost.storageAtRest 0.0208000000001) (eq .root.Values.global.cost.storageReads 0.00400000000001)) -}}
+ {{- if or (eq (float64 .root.Values.global.cost.storageWrites) 0.0500000000001) (or (eq (float64 .root.Values.global.cost.storageAtRest) 0.0208000000001) (eq (float64 .root.Values.global.cost.storageReads) 0.00400000000001)) -}}
```

Comment:
Change makes sense, but floating point quality tests are not recommended because of potential differences. in the last bits
would be better code checking a certain number of significant digits e.g. something like 
(int)((double)value * 1000000)) == 50000

Reply:
let's merge this one, and change after.
@shamser - can you look at making these changes?